home_task_queries.sql

```diff
@@ -26,23 +37,41 @@
 Вивести категорія фільмів, на яку було витрачено найбільше грошей
 в прокаті
 */
--- SQL code goes here...
 
+select c.name, sum(p.amount) as amount from category as c
+    join film_category fc on c.category_id = fc.category_id
+    join inventory i on fc.film_id = i.film_id
+    join rental r on i.inventory_id = r.inventory_id
+    join payment p on r.rental_id = p.rental_id
+group by c.name
+order by amount desc
+limit 1
 
 
 /*
 4.
 Вивести назви фільмів, яких не має в inventory.
 Запит має бути без оператора IN
 */
--- SQL code goes here...
+select f.title, i.film_id from film as f
+    left join inventory i on f.film_id = i.film_id
+                            where i.film_id is null
+group by f.film_id, i.film_id
+order by f.film_id
+
 
 
 /*
 5.
 Вивести топ 3 актори, які найбільше зʼявлялись в категорії фільмів “Children”.
 */
--- SQL code goes here...
+select a.first_name, a.last_name, count(1) as count from category as c
+    inner join film_category fc on c.category_id = fc.category_id and c.name = 'Children'
+    inner join film_actor fa on fc.film_id = fa.film_id
+    inner join actor a on a.actor_id = fa.actor_id
+group by a.first_name, a.last_name
```

Comment:
if you're not sure you have a few actors with the same first and last name, you should group by actor_id